specification/attestation/data-plane/Microsoft.Attestation/stable/2022-08-01/examples/AttestSgxEnclave.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Why this attest sevsnp example is different from the example in this PR?https://dev.azure.com/msazure/One/_git/security-Attestation/pullrequest/6780314?_a=files&path=/src/AttestationServices/RestApi/ReadOnlyGeneratedSwaggerFiles/attestation/data-plane/Microsoft.Attestation/stable/2022-08-01/examples/AttestSevSnpVm.json

Reply:
I believe you are looking at the attest SGX enclave example..